packages/gatsby-theme-bulmaio/src/references/gatsby-setup.ts

```diff
@@ -21,7 +21,7 @@ async function createListing(graphql: any, createPage: any, referenceType: strin
     }
   `);
     const allResults = allResultsData.data[allType].nodes;
-    const entriesPerPage = 200;  // TODO Lower me
+    const entriesPerPage = 25;
```

Comment:
I have just put `25`, do you think this is OK? Do we need a lower/higher number of items per page?

Reply:
25 might still be too high. Lots of big images killing our page load size.